ipa-core/src/report.rs

```diff
@@ -425,6 +626,8 @@ where
     pub breakdown_key: Replicated<BK>,
     pub trigger_value: Replicated<TV>,
     pub timestamp: Replicated<TS>,
+    pub epoch: Epoch,
+    pub site_domain: String,
```

Comment:
I would be cautious about site domain as well. We are adding it here as owned value but it is very likely that for real inputs there will be a lot of events for the same site domain. Adding lifetimes to the report struct may be very hard later. 

I wonder if we should just have a hashset as string interner and let report borrow from it? Not necessarily in this PR but if we want to do it, we should do it sooner than later

Reply:
I think Its best to remove it for now and once we include it in a PR, we can think of how to best include epoch and site_domain. Probably epoch will be included in timestamp.

Reply:
Maybe we should also separate the struct into two structs by adding OPRFInputRow. Then we would not need to serialize and deserialize epoch and site domain.

Reply:
this struct comes into play when the report has been decrypted already, so we shouldn't have a need for things related to crypto. Separating data required to perform MPC computation and everything else is important I think. 

It is likely because we use the same struct on the client and server side and it is really unnecessary imo, especially because there is a cost we have to pay on the server side for it. So it deserves at least a comment/issue so we can come back to it later.

Reply:
Yes, I agree that it would make sense to separate report and input. There are several issue related to it. Initially Richa used the Report struct as input for the MPC computation. I was building on top of it. If we want to separate it, we probably need to remove the OprfReport struct from all the plumbing in the MPC process and replace it with something like OprfInput.

Reply:
I have split the struct into OprfReport and OPRFIPAInputRow